src/sequelize.ts

```diff
@@ -1,6 +1,6 @@
 import { Sequelize } from 'sequelize-typescript'
 
-const env = process.env.NODE_ENV || 'development'
+const env = process.env.DB_ENV || process.env.NODE_ENV || 'development'
```

Comment:
Haven't seen `DB_ENV` used before. When does this get set instead of `NODE_ENV`?

Reply:
It's just for CI since the DB connection is slightly different for CI vs local test. I'll add a comment to clarify.

Reply:
Ah got it. Sounds good 👍